### DISCUSSION TITLE
Add an option to make a photo private straight from the app, just like archive or deleting.

### DISCUSSION THREAD

**Discussion by RogueGhost93** [Radiokot/photoprism-android-client#145] [Ideas]
👉🏻 [Start discussing an idea](https://github.com/Radiokot/photoprism-android-client/discussions/new?category=ideas)

Slideshow is a great way to see what photos should not bee seen again for whatever reason, making it possible to make them private straight from slideshow would be amazing! Thanks to anyone who is willing to try to implement it!

**Comment by Radiokot** [comment]
Hi, thanks, it's a good idea. I doubt it will be added to slideshow, but another button like "Archive" or "Delete" could be added.
There is also an idea to implement a Tinder-like review process, where you go through the list of photos by swiping them left to hide/discard or right to show/accept.

**Comment by andrejserafim** [comment]
On mobile this I only switch to the web UI to make a photo private. So a button in the app would be super-awesome!

I use the private flag to hide pictures I still need, but for other reasons than my family photos.

**Comment by Radiokot** [comment]
It's a good idea, I'll add it soon

**Comment by Radiokot** [comment]
Added in today's update: https://github.com/Radiokot/photoprism-android-client/releases/tag/1.32.0(48)

**Reply by andrejserafim** [reply]
works like a charm, thank you!
